setup.py

```diff
@@ -12,6 +12,9 @@
 
 from setuptools import setup
 
+readme = open('README.rst').read()
+history = open('CHANGES.rst').read()
```

Comment:
Please add also `docs/requirements.txt` for RTFD with any necessary dependencies, e.g.

```
-e .[all]
```

is usually enough.